Hush/Assets/Scripts/Player/PlayerInputManager.cs

```diff
@@ -46,21 +46,25 @@ public void OnLook(CallbackContext context)
 	public void OnLightAttack(CallbackContext context)
 	{
 		lightAttack = context.ReadValueAsButton();
+		GameMaster.IsEngagedInCombat(context.ReadValueAsButton());
```

Comment:
Remove IsEngagedinCombat for this and further inputs

Reply:
done in f3c60de